packages/analytics-js/src/components/userSessionManager/types.ts

```diff
@@ -21,3 +21,8 @@ export interface IUserSessionManager {
   end(): void;
   clearUserSessionStorage(resetAnonymousId?: boolean): void;
 }
+
+export type SessionTrackingInfo = {
+  id?: number;
```

Comment:
Why is this field optional?

Reply:
In case auto/manual tracking is not enabled.

Reply:
If we create an instance of this type, we'll always have value for `id`.

Reply:
So what do you suggest this fn should return in case both the session tracking is disabled?

Reply:
Which function?

Reply:
getSessionInfo